sql/core/src/test/scala/org/apache/spark/sql/SubquerySuite.scala

```diff
@@ -893,7 +893,7 @@ class SubquerySuite extends QueryTest with SharedSQLContext {
           """
             |SELECT * FROM t1
             |WHERE
-            |NOT EXISTS (SELECT * FROM t1)
+            |NOT EXISTS (SELECT * FROM t1 AS t2 WHERE t2.i = t1.i)
```

Comment:
I changed this test because it focuses on correlated subquery which will rewrite as join.